_test_data/proto-AND--p-0.1--net-toy_network.csv

```diff
@@ -0,0 +1,34 @@
+,actor_id,simulation_length,actors_infect,actors_ninfect,peak_infect_nb,peak_infect_iter
```

Comment:
I think we should keep a consistent output format between this package and the influence-simulator.

Another thing - skip the row_id (especially that it doesn't even have a column name) it adds no additional info

Reply:
I've updated both the column naming and an index in csv, but honestly it doesn't matter for me wheatear I store the index or not